resources/js/package.js

```diff
@@ -157,6 +157,10 @@ function init() {
                 canOrder() {
                     return this.cart.items.every((item) => item.is_available)
                 },
+
+                queryParams() {
+                    return new URLSearchParams(window.location.search)
```

Comment:
This is reactive? It changes as we type?

Reply:
Nevermind, not needed. From InstantSearch it will be reactive, see: https://github.com/rapidez/core/pull/799